src/components/Pagination/Pagination.story.js

```diff
@@ -17,6 +17,11 @@ storiesOf(`${GROUPS.COMPONENTS}|Pagination`, module)
         page={number('page', 1)}
         perPage={number('perPage', 10)}
         total={number('total', 50)}
+        pagesToShow={number('pagesToShow', 1)}
+        onChange={() => {}}
+        footer={text('footer', 'Showing 1 - 10 of 50 items')}
+        align={text('align', 'center')}
+        justify={text('justify', 'center')}
```

Comment:
Please use the [`select` knob](https://www.npmjs.com/package/@storybook/addon-knobs#select) here, so users can try out the available options.

Reply:
If we firmly decide to use strictly the CSS options, then it makes sense for this to be predefined options.